hbase-server/src/test/java/org/apache/hadoop/hbase/HBaseTestingUtil.java

```diff
@@ -3351,139 +3342,6 @@ public static void waitForHostPort(String host, int port) throws IOException {
     }
   }
 
-  /**
```

Comment:
Just hoping it is fine to move code from here as class is marked @InterfaceStability.Evolving. Otherwise may have to think some other approach.